File: python/Cinema.py
class Cinema:
    def __init__(self, id="", title="", genre="", released=0, duration=0):
        self._id = id
        self._title = title
        self._genre = genre
        self._released = released
        self._duration = duration

    # Getter Method
    def get_id(self):
        return self._id

    def get_title(self):
        return self._title

    def get_genre(self):
        return self._genre

    def get_released(self):
        return self._released

    def get_duration(self):
        return self._duration

    # Setter Method
    def set_id(self, id):
        self._id = id

    def set_title(self, title):
        self._title = title

    def set_genre(self, genre):
        self._genre = genre

    def set_released(self, released):
        self._released = released

    def set_duration(self, duration):
        self._duration = duration

    # Data satu objek jadi satu baris untuk tabel
    def to_row(self):
        return [self._id, self._title, self._genre,
                f"{self._duration} menit", str(self._released)]

    def __str__(self):
        return f"Cinema({self._id}, {self._title}, {self._genre}, {self._released}, {self._duration})"

File: java/Cinema.java
import java.util.List;
import java.util.Arrays;

public class Cinema {
    private String id;
    private String title;
    private String genre;
    private int released;
    private int duration;

    // Konstruktor kosong
    public Cinema() {}

    // Konstruktor dengan parameter
    public Cinema(String id, String title, String genre, int released, int duration) {
        this.id = id;
        this.title = title;
        this.genre = genre;
        this.released = released;
        this.duration = duration;
    }

    // Getter Method
    public String getId() { return id; }
    public String getTitle() { return title; }
    public String getGenre() { return genre; }
    public int getReleased() { return released; }
    public int getDuration() { return duration; }

    // Setter Method
    public void setId(String id) { this.id = id; }
    public void setTitle(String title) { this.title = title; }
    public void setGenre(String genre) { this.genre = genre; }
    public void setReleased(int released) { this.released = released; }
    public void setDuration(int duration) { this.duration = duration; }

    // Data satu objek jadi satu baris untuk tabel
    public List<String> toRow() {
        return Arrays.asList(id, title, genre, duration + " menit", String.valueOf(released));
    }
}